common/openstack/adding-an-openstack-infrastructure-provider.adoc

```diff
@@ -0,0 +1,57 @@
+After initial installation and creation of a {product-title} environment, add an OpenStack Infrastructure provider to the appliance. {product-title} supports operating with the OpenStack `admin` tenant.
```

Comment:
Please edit:
I don't think Infrastructure should be capitalized, or it looks like "OpenStack Infrastructure" is the product's proper name. "OpenStack infrastructure provider" I think is better, as it's an OpenStack provider, and infrastructure is the provider type.